src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -54,9 +54,18 @@ public Transaction BuildTransaction(TransactionBuildContext context)
             if (context.Shuffle)
                 context.TransactionBuilder.Shuffle();
 
-            Transaction transaction = context.TransactionBuilder.BuildTransaction(context.Sign);
+            bool validationSuccess = false;
+            Transaction transaction = null;
+            TransactionPolicyError[] errors = null;
 
-            if (context.TransactionBuilder.Verify(transaction, out TransactionPolicyError[] errors))
+            var retryOptions = new RetryOptions(3, TimeSpan.FromMilliseconds(5), RetryStrategyType.Simple, typeof(ArgumentNullException));
```

Comment:
is delay time important here?

Reply:
@MithrilMan This is just part of the architecture at the moment. I suppose it is not that important but the method does not currently allow to not specify a delay unless you use the default constructor.